.github/workflows/compatibility.yaml

```diff
@@ -40,6 +40,8 @@ jobs:
           ./telepresence-server version
 
           ./telepresence-server helm install
+          # Quit the user and root daemons since they get started by the installation.
+          ./telepresence-server quit
```

Comment:
A `telepresence quit` doesn't quit the daemons. It just quits the current session.  Use `-s` to ensure that the daemons really die.

Reply:
@thallgren You probably figured this out, but the more recent version does what you suggest. I should have either rebased or mentioned that the history is a bit dirty due to the usual CI debug dance and it's better to review the full diff.